config/src/main/java/org/hyperledger/besu/config/GenesisConfigFile.java

```diff
@@ -75,6 +79,25 @@ public static GenesisConfigFile fromConfig(final ObjectNode config) {
     return new GenesisConfigFile(normalizeKeys(config));
   }
 
+  public static GenesisConfigOptions getMainnetConfigOptions() {
```

Comment:
This and constant is supports is inessential to Typed Transactions and should be in it's own PR.

Reply:
Should this be deleted?  Didn't it get moved here - https://github.com/hyperledger/besu/pull/1688/files#diff-df7c51531e08d1f892222e589588a0f67d907b10d6f4d84f6427eacb46ee4a63R38?